### DISCUSSION TITLE
"interpolation decay" not working with current "component import"

### DISCUSSION THREAD

**Discussion by Aztec91** [google/model-viewer#2333] [Q&A]
Hey I just couldnt figure out how to use the "interpolation-decay" attribute:

with: 

<script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>

using:

<script type="module" src="https://modelviewer.dev/node_modules/@google/model-viewer/dist/model-viewer.min.js"></script>

it worked tho.

Will this be fixed soon?

kind regards
Aztec

**Comment by ghost** [comment]
Yes, this is expected the examples are served off of master, this feature
has not yet been officially released.

**Reply by Aztec91** [reply]
Thank you Very much for the Feedback.

**Reply by LaloYanez** [reply]
If you really need it you can manually set it. You could branch the code and look for `const DECAY_MILLISECONDS`. The value is 50, you can play with that to increase or decrease the speed.

When the feature is released I believe we will be able to manipulate it on the fly!
